rpcs3/Emu/RSX/GL/GLProgramBuffer.h

```diff
@@ -36,8 +36,8 @@ struct GLTraits
 	{
 		pipeline_storage_type result = std::make_unique<gl::glsl::program>();
 		result->create()
-			.attach(gl::glsl::shader_view(vertexProgramData.id))
-			.attach(gl::glsl::shader_view(fragmentProgramData.id))
+			.attach(gl::glsl::shader_view(vertexProgramData.shader.id()))
```

Comment:
The id member should still exist to generate the pipeline keys. This decouples the cache from the backend implementation, we shouldn't have to change this code for example if the GLVertexProgram/GLFragmentProgram internals change again.

Reply:
Ah yes it still exists, it can be accessed through just .id as well, I'll change it back